addons/website_slides/tests/test_slide_channel.py

```diff
@@ -370,3 +370,38 @@ def test_channel_enroll_policy(self):
 
         copied_channel = channel.copy()
         self.assertEqual(copied_channel.enroll, 'invite', "Copied channel should have the same enroll field value")
+
+    def test_invite_email_translation(self):
```

Comment:
Side note: invite tests are inside 'test_attendee.py' test file (see notably 'test_invite_to_course'), so would add the test in that file to keep them in the same file :)